app/services/widgets/settings/alert-box/index.ts

```diff
@@ -211,6 +212,10 @@ export class AlertBoxService extends WidgetSettingsService<IAlertBoxData> {
       setting.image_href === '/images/gallery/default.gif'
         ? 'http://uploads.twitchalerts.com/image-defaults/1n9bK4w.gif'
         : setting.image_href;
+    const constrainedDuration =
```

Comment:
Can we add comments about this logic, or extract consts? Magic numbers aren't always clear on what they meant or why they were chosen. I'd assume `1000` is ms to sec, but even that might be wrong, the `300` is even more obscure.

Reply:
300 is the max duration for alerts, 1000 is ms to sec yea

Reply:
1000 is ms to sec yea, 300 seconds is the maximum duration for alerts. would really prefer to have things like this handled on the backend but this is the api we inherited